packages/data-stores/src/plans/hooks/use-pricing-meta-for-grid-plans.ts

```diff
@@ -56,6 +56,12 @@ interface Props {
 	 * If true, the pricing includes discounts from upgrade credits.
 	 */
 	withProratedDiscounts?: boolean;
+
+	/**
+	 * TODO: Add more detailed comments because we've decided to make a tempoarary decision around storage-addons
+	 * while the calypso_plans_page_emphasize_longer_plan_savings experiment is running.
+	 */
+	enableTermSavingsPriceDisplay?: boolean;
```

Comment:
Taking a closer look after I test and review https://github.com/Automattic/wp-calypso/pull/96492

Reply:
> Although, on second thought, it might be the case we may actually want to query AddOns.useStorageAddOns hook internally here (instead of passing in as a prop), which would make this control property more appropriate. This would be a nice cleanup/refactor on top if we stick with the new prop enableTermSavingsPriceDisplay (see my other comment though).

This makes sense to me if storage add-ons are an integral part of the pricing meta grid for plans. I think you'll have a better sense of this since you work so closely with the grid, but it sounds like we've reached that point 😎

If that's the case, yeah we can definitely stick with and rename the `enableTermSavingsPriceDisplay` prop

Reply:
> Although, on second thought, it might be the case we may actually want to query AddOns.useStorageAddOns hook internally here (instead of passing in as a prop), which would make this control property more appropriate. This would be a nice cleanup/refactor on top if we stick with the new prop enableTermSavingsPriceDisplay (see my other comment though).

Starting to work more on this in a follow-up